### DISCUSSION TITLE
renovate with yarn.lock

### DISCUSSION THREAD

**Discussion by Rahul-learn** [renovatebot/renovate#15995] [Request Help]
### How are you running Renovate?

Self-hosted

### Please select which platform you are using if self-hosting.

GitHub Enterprise Server

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

We are having issue while updating dependency for yarn.lock through renovate. We are using renovatebot to update dependency for maven,python which is working fine but we have a repo with yarn.lock which is causing issue, Renovate is creating a PR but the same time its adding the below comment to all the renovate creating PR's for yarn.lock file

warning package.json: No license field
warning @cep/react-example@0.1.0: No license field
warning Resolution field "minimist@1.2.6" is incompatible with requested version "minimist@0.0.8"
warning Resolution field "jake@0.1.22" is incompatible with requested version "jake@^10.8.5"
error An unexpected error occurred: "https://nexus.com/repository/ssri: unable to get local issuer certificate".

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by viceice** [comment]
You're using self-signed certificates? You need to configure renovate and tools to accept them

**Reply by Rahul-learn** [reply]
I am setting NODE_EXTRA_CA_CERTS certs in my docker file. Do I need to set any other variable  specifically for yarn because renovate is working for other managers (maven,package.json,requiremtn.txt)?

ENV NODE_EXTRA_CA_CERTS=/certs.pem

**Reply by viceice** [reply]
which `binary-source` are you using?

**Reply by Rahul-learn** [reply]
Using nexus repository manger

**Comment by rarkins** [comment]
Future issues will be deleted without explanation - please raise only discussions

**Comment by PhilipAbed** [comment]
Hi Rahul,

i have no way to reproduce/investigate since there's no debug logs, but i tried to look for the origin of the error

Are you running behind a proxy? ,
it seems we are trying to run yarn install in order to update your yarn.lock, but it's failing with this error :
https://stackoverflow.com/questions/40124008/error-unable-to-get-local-issuer-certificate-while-running-yarn-command

The solution could be in this stack-overflow URL above, all you need is for renovate to be able to run yarn install without failing and your issue will be resolved.

i cant see a way to investigate any further with the amount of information we have, but hope this helps.

if this didn't work try reading here for solutions:
https://github.com/yarnpkg/yarn/issues/841
